frontend/apps/web/app/(mgmt)/[account]/transformers/EditTransformerOptions.tsx

```diff
@@ -93,7 +95,21 @@ export default function EditTransformerOptions(props: Props): ReactElement {
                   </Badge>
                 </div>
               </div>
-              <DialogDescription>{transformer?.description}</DialogDescription>
+              <div className="flex flex-row items-center gap-2">
+                <DialogDescription>
+                  {transformer?.description}{' '}
+                  {/* don't render the learn more link for the js transformers since they have their own special link && only render for system transformers*/}
+                  {TransformerSource[transformer.source] !=
+                    'GENERATE_JAVASCRIPT' &&
+                    TransformerSource[transformer.source] !=
+                      'TRANSFORM_JAVASCRIPT' &&
+                    isSystemTransformer(transformer) && (
```

Comment:
would be great if this could be pulled into a function or something since it's now used in a few spots.

Also, when comparing the transformer sources, you can just use the importable enumeration instead of doing string comparisons.
`transformer.source !== TransformerSource.GENERATE_JAVASCRIPT` etc.